spk/salt-master/src/requirements.txt

```diff
@@ -1,13 +1,23 @@
-salt==2015.5.11
-Jinja2==2.11.2
-requests==2.23.0
-CherryPy==11.2.0
-more-itertools==5.0.0
-cheroot==6.3.3
-backports.functools-lru-cache==1.4
-jaraco.windows==3.7
-jaraco.classes==1.4.3
-jaraco.functools==1.17
-portend==2.2
-tempora==1.10
-futures==3.3.0
+certifi==2021.5.30
+charset-normalizer==2.0.4
+contextvars
+distro>=1.0.1
+idna==3.2
+immutables==0.16
+Jinja2
+MarkupSafe
+msgpack>=0.5,!=0.5.5
+# Avoid 3.10.1 since it has 
+# [bdist_wheel]
+# py-limited-api = cp35
+pycryptodomex>=3.9.8,!=3.10.1
+python-dateutil==2.8.2
+PyYAML==5.4.1
+pyzmq<=20.0.0 ; python_version < "3.6"
+pyzmq>=17.0.0 ; "arm" not in platform_machine
+pyzmq==17.0.0 ; "arm" in platform_machine
+pyzmq>=19.0.2 ; python_version >= "3.9"
+requests>=1.0.0
+salt==3003.3
```

Comment:
```suggestion
```

I recon this is why. The patch is overridden by installing salt on top of it

Reply:
Thanks for pointing out that salt-minion has the patch otherwise I would not have found it.